src/mdxast-mermaid.ts

```diff
@@ -30,7 +30,7 @@ function plugin (config?: Config) {
     // See if there is already an import for the Mermaid component
     let importFound = false
     visit(ast, { type: 'import' }, (node: Literal<string>) => {
-      if (/\s*import\s*{\s*Mermaid\s*}\s*from\s*'mdx-mermaid\/Mermaid'\s*;?\s*/.test(node.value)) {
+      if (/\s*import\s*{\s*Mermaid\s*}\s*from\s*'mdx-mermaid\/lib\/Mermaid'\s*;?\s*/.test(node.value)) {
```

Comment:
Can the `lib\/` be made optional so that it is compatible with `mdx-mermaid/Mermaid` imports

Reply:
> Can the `lib\/` be made optional so that it is compatible with `mdx-mermaid/Mermaid` imports
Yeah, I've done that and added a new test case.😄